instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-common/javaagent/src/test/groovy/JaxRsClientTest.groovy

```diff
@@ -81,6 +103,11 @@ abstract class JaxRsClientTest extends HttpClientTest implements AgentTestTrait
     "/client-error" | 400
     "/error"        | 500
   }
+
+  @Override
+  boolean testAsync() {
+    false
+  }
```

Comment:
I see doRequestAsync above
```suggestion
```

Reply:
Ah yeah I was surprised to find the async pattern for a seemingly old library :)